plugin-server/src/main/graphile-worker/worker-setup.ts

```diff
@@ -0,0 +1,111 @@
+import Piscina from '@posthog/piscina'
+import { CronItem, TaskList } from 'graphile-worker'
+
+import { EnqueuedBufferJob, EnqueuedPluginJob, Hub } from '../../types'
+import { status } from '../../utils/status'
+import { pauseQueueIfWorkerFull } from '../ingestion-queues/queue'
+import { runInstrumentedFunction } from '../utils'
+import { runBufferEventPipeline } from './buffer'
+import { loadPluginSchedule, runScheduledTasks } from './schedule'
+
+export async function startGraphileWorker(hub: Hub, piscina: Piscina): Promise<Error | undefined> {
+    status.info('🔄', 'Starting Graphile Worker...')
+
+    let jobHandlers: TaskList = {}
+
+    const crontab: CronItem[] = []
+
+    if (hub.capabilities.ingestion) {
```

Comment:
@hazzadous you're not going to like that I've moved this back out from `pluginsServer` but it makes it much easier/clear to test, particularly when it comes to capabilities testing

Reply:
Hah well as long as we can test with the e2e that we are correct I'm fine with that.